docs/src/prototypes/chatPane/chatPaneContent.tsx

```diff
@@ -26,12 +26,20 @@ class ChatPaneContainer extends React.PureComponent<ChatPaneContainerProps> {
     return (
       items.length > 0 && (
         <Scrollbars ref={this.handleScrollRef}>
-          <Chat
-            items={items}
+          <div
             role="main"
-            aria-label={`${chat.title} chat content.`}
-            styles={{ padding: '0 32px' }}
-          />
+            aria-label="Message list. In forms mode: press Enter to explore message content, then use Escape to shift focus back to the message"
+          >
+            <div
+              id="chat-pane-reader-text"
+              style={screenReaderMessageContainerStyles}
+              role="heading"
+              aria-level={2}
+            >
+              Message list.
```

Comment:
Is it enough descriptive to have just "Message list."? Previously, we used chat's title in naming, like `${chat.title} chat content.`

Reply:
I was following documentation with naming "Message list". 
The instruction are currently on the "main" element, as we didn't figure out anything better. In old code we set instruction on every single message, what was annoying for users. 
But there will be follow up discussion about instruction.